device/arista/x86_64-arista_7060x6_64de/Arista-7060X6-64DE-O128S2/th5-a7060x6-64de.config.bcm

```diff
@@ -0,0 +1,1416 @@
+#
+# $Copyright: (c) 2022 Broadcom.
+# Broadcom Proprietary and Confidential. All rights reserved.$
+#
+# BCM78900 64x800g port configuration.
```

Comment:
hi Chris, I didn't see any UC/MC queue settings as well as buffer initialization in the BCM file, will the switch still be able to perform the L2/L3 forwarding with this configuration?

Reply:
Are you seeing any issues with L2/L3 forwarding?

Reply:
yea, David. The L2 MC would not work from what we see in Tristan. Let's get this in first. We can create another BCM case to get the BCM config.